osfoffline/database/models.py

```diff
@@ -81,6 +105,14 @@ def validate_top_level(self, key, top_level):
             assert self.parent is not None
         return top_level
 
+    @validates('sync')
+    def validate_sync(self, key, sync):
+        if not self.parent_id:
+            assert sync is True
```

Comment:
As long as it's only internal and for debugging we can work with that; thanks.

The reason I brought this up at all is that we've been trying to move away from the existing convention in some legacy code, actually. :) It's an ongoing process...